actions/gha-workflow-validator/jest.config.ts

```diff
@@ -0,0 +1,13 @@
+import {Config} from 'jest'
+export default {
+  displayName: "gha-workflow-validator",
+  preset: "../../jest.preset.js",
+  testEnvironment: "node",
+  transform: {
+    "^.+\\.[tj]s$": "@swc/jest",
+  },
+  moduleFileExtensions: ["ts", "js", "html"],
```

Comment:
```suggestion
  moduleFileExtensions: ["ts"],
```
Can we remove these?

Reply:
I can remove `html`

```
  >  NX   ● Validation Error:

   
     moduleFileExtensions must include 'js':
     but instead received:
       ["ts"]
     Please change your configuration to include 'js'.
   
     Configuration Documentation:
     https://jestjs.io/docs/configuration
```